guides/source/getting_started.md

```diff
@@ -113,6 +122,47 @@ $ sqlite3 --version
 
 The program should report its version.
 
+#### Installing Node
+
+Since Rails comes with Webpacker as JavaScript pre-processor and bundler,
+you will need to have Node runtime installed on your machine. Webpacker requires Node
+version 8.16.0 or later. Find the installation
+instructions at the [Node website](https://nodejs.org/en/download/) and 
+verify it's installed correctly with the following command:
+
+```bash 
+$ node --version
+```
+
+The version of your Node runtime should be printed out. Make sure it's greater
+than 8.16.0.
+
+You should also have NPM installed, so make sure this command prints out
+the version:
+
+```bash
+$ npm --version
+```
+
+#### Installing Yarn
+
+Since Webpacker requires Yarn as a package manager, make sure you have
+it installed on your machine. If you don't, run this command to install it:
+
+```bash
+$ npm install -g yarn
```

Comment:
Installing Yarn via npm is discouraged:

https://classic.yarnpkg.com/en/docs/install/#install-via-npm

> **Note:** Installation of Yarn via npm is generally not recommended. When installing Yarn with Node-based package managers, the package is not signed, and the only integrity check performed is a basic SHA1 hash, which is a security risk when installing system-wide apps.
>
> For these reasons, it is highly recommended that you install Yarn through the installation method best suited to your operating system.

Perhaps we should link to that page instead, so that users can follow the OS-specific instructions?

Reply:
Good catch, thanks!